src/nav/MainNavBar.js

```diff
@@ -28,22 +44,44 @@ export default class MainNavBar extends React.Component {
     }
 
     return (
-      <Navigator
-        initialRoute={{ name: 'Home', index: 0 }}
-        renderScene={(route) =>
-          <View style={moreStyles.wrapper}>
-            <StatusBar
-              barStyle={textColor === 'white' ? 'light-content' : 'dark-content'}
-            />
-            <View style={[styles.navBar, { backgroundColor }]}>
-              <NavButton name="ios-menu" color={textColor} onPress={onPressStreams} />
-              <Title color={textColor} />
-              <NavButton name="md-people" color={textColor} onPress={onPressPeople} />
+      <MenuContext
```

Comment:
Same comment as above. This stuff should go into its own component and not be coupled to the MainNavBar